src/Text/Pandoc/Readers/Org.hs

```diff
@@ -472,7 +483,7 @@ rawBlockContent (indent, blockType) = try $
   unlines . map commaEscaped <$> manyTill indentedLine blockEnder
  where
    indentedLine = try $ ("" <$ blankline) <|> (indentWith indent *> anyLine)
-   blockEnder = try $ indentWith indent *> stringAnyCase ("#+end_" <> blockType)
+   blockEnder = try $ skipSpaces *> stringAnyCase ("#+end_" <> blockType)
```

Comment:
+++ Albert Krewinkel [Apr 25 16 06:41 ]:
>   I think Emacs is doing it wrong here, so I'd rather not change it
>   unless there are any good arguments to do so.

Shouldn't we be trying to match Emacs' behavior?  After all,
that is the authoritative standard for org-mode.  One might
have views about how org-mode could have been designed
better, but unless you think Emacs' behavior is actually
buggy (relative to the org-mode documentation), I think
departing from it is a bad idea.